flexmeasures/cli/utils.py

```diff
@@ -28,6 +28,66 @@ class MsgStyle(object):
     ERROR: dict[str, Any] = {"fg": "red"}
 
 
+class DeprecatedOption(click.Option):
```

Comment:
These two classes should both mention [where you got them from](https://stackoverflow.com/a/50402799/13775459), and please mention whether you adapted them.

If adapted:
```
"""<existing docstring contents>

References
----------------

Adapted from https://stackoverflow.com/a/50402799/13775459
"""
```
Else if copied verbatim:
```
"""<existing docstring contents>

References
----------------

Copied from https://stackoverflow.com/a/50402799/13775459
"""
```

Please note that contents from StackOverflow come with a licence (nowadays, [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)) that demands attribution.